securedrop/sdconfig.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+import typing
+# https://www.python.org/dev/peps/pep-0484/#runtime-or-type-checking
+if typing.TYPE_CHECKING:
+    # flake8 can not understand type annotation yet.
+    # That is why all type annotation relative import
+    # statements has to be marked as noqa.
+    # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
+    from typing import List, Dict  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        try:
+            self.FlaskConfig = _config.FlaskConfig  # type: ignore
+        except AttributeError:
+            pass
+
+        try:
+            self.JournalistInterfaceFlaskConfig = \
```

Comment:
please do not include modules, only variables

Reply:
https://github.com/freedomofpress/securedrop/blob/develop/securedrop/journalist_app/__init__.py#L29

@dachary That is not a module, it is class which is part of current `config.py`.

Reply:
correct, we need to keep that